SConstruct

```diff
@@ -1028,17 +1029,19 @@ if "SAVE_OPTIONS" in ARGUMENTS :
 if env["BUILD_CACHEDIR"] != "" :
 	CacheDir( env["BUILD_CACHEDIR"] )
 
+env["IECORE_MILESTONE_VERSION"] = ieCoreMilestoneVersion
```

Comment:
I'd initially changed all of these to `CORTEX_` and then bailed out when I realized there are many other variables/options that should probably be renamed as well...